src/transformers/__init__.py

```diff
@@ -4081,6 +4085,9 @@
         logging,
     )
 
+    # bitsandbytes config
+    from .utils.quantization_config import BitsAndBytesConfig
```

Comment:
Is this the right approach? If I protect this import with `is_bitsandbytes_available` the docs won't be able to build it seems

Reply:
Yes, especially since all bnb imports are protected, so you can import this directly here.